src/react_native/reanimated.cljs

```diff
@@ -57,11 +57,13 @@
 ;; Helper functions
 (defn get-shared-value
   [anim]
-  (.-value anim))
+  (when anim
+    (.-value anim)))
 
 (defn set-shared-value
   [anim val]
-  (set! (.-value anim) val))
+  (when anim
```

Comment:
When `navigate-to-chat-nav2` is called, we also change the selected tab by modifying the shared value. But in integration tests, shared values are not initialized and accessing the -.value of nil throws an error